flixel/graphics/tile/FlxDrawTrianglesItem.hx

```diff
@@ -137,17 +141,24 @@ class FlxDrawTrianglesItem extends FlxDrawBaseItem<FlxDrawTrianglesItem>
 	}
 
 	public function addTriangles(vertices:DrawData<Float>, indices:DrawData<Int>, uvtData:DrawData<Float>, ?colors:DrawData<Int>, ?position:FlxPoint,
+			angle:Float = 0, ?scale:FlxPoint, ?originPoint:FlxPoint,
```

Comment:
adding these arguments breaks flash via FlxClothSprite calls this and flash cannot skip the `angle` arg